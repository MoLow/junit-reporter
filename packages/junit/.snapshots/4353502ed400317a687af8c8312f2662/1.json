{
  "stderr": "",
  "stdout": "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n<testsuites>\n\t<testcase name=\"should fail\" time=\"*\" classname=\"test\" failure=\"false == true\">\n\t\t<failure type=\"testCodeFailure\" message=\"false == true\">\n[Error [ERR_TEST_FAILURE]: false == true] {\n  failureType: 'testCodeFailure',\n  cause: AssertionError [ERR_ASSERTION]: false == true\n      *\n      *\n      *\n      *\n      * {\n    generatedMessage: true,\n    code: 'ERR_ASSERTION',\n    actual: false,\n    expected: true,\n    operator: '=='\n  },\n  code: 'ERR_TEST_FAILURE'\n}\n\t\t</failure>\n\t</testcase>\n\t<testcase name=\"should pass\" time=\"*\" classname=\"test\"/>\n</testsuites>\n",
  "exitCode": 1
}