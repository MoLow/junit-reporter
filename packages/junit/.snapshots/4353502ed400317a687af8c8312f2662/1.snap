Object {
  "exitCode": 1,
  "stderr": "",
  "stdout": "<?xml version=\"1.0\" encoding=\"utf-8\"?>
<testsuites>
	<testcase name=\"should fail\" time=\"*\" classname=\"test\" failure=\"false == true\">
		<failure type=\"testCodeFailure\" message=\"false == true\">
[Error [ERR_TEST_FAILURE]: false == true] {
  failureType: 'testCodeFailure',
  cause: AssertionError [ERR_ASSERTION]: false == true
      *
      *
      *
      *
      * {
    generatedMessage: true,
    code: 'ERR_ASSERTION',
    actual: false,
    expected: true,
    operator: '=='
  },
  code: 'ERR_TEST_FAILURE'
}
		</failure>
	</testcase>
	<testcase name=\"should pass\" time=\"*\" classname=\"test\"/>
</testsuites>
",
}